site/cypress/e2e/Cypress-UI/navigation_page.spec.js

```diff
@@ -9,6 +7,91 @@ function validate_navigation_page_sections(sections) {
     });
 }
 
+const white = 'rgb(255, 255, 255)';
+const blue = 'rgb(0, 99, 152)';
+const red = 'rgb(220, 53, 69)';
+const green = 'rgb(95, 183, 96)';
+const buttonOrder = ['team-btn', 'submit-btn', 'grade-btn', 'quick-link-btn'];
+
+function checkButtons(courseID, buttonText, buttonColor) {
```

Comment:
Not a required change, but it would be easier to read if buttonText and buttonColor was plural, or otherwise a bit more descriptive that it's the list of button texts, or colors. I had a hard time understanding it at first.

Reply:
Is the arrays in the test easy to understand, or what would be your recommendation

Reply:
See below review, they are easy to understand once you read through it, its just a bit cluttered.